### DISCUSSION TITLE
Overriding harvester configuration changed in 1.4

### DISCUSSION THREAD

**Discussion by lauriseitsonen** [EOEPCA/eoepca#112] [Data Access] [answered]
Harvester documentation at
https://deployment-guide.docs.eoepca.org/current/eoepca/data-access/#starting-the-harvester instructs to use `--values` switch with json data to override default values. This switch is not supported with the harvester that ships with v1.4. Instead switch `-co` (or `--config-override`) should be used, e.g. 
```
kubectl -n rm exec -it deployment.apps/data-access-harvester -- \
python3 -m harvester harvest --config-file /config.yaml Sentinel2 \
-co harvesters.Sentinel2.resource.opensearch.query.time.begin=2024-01-01T00:00:00Z \
-co harvesters.Sentinel2.resource.opensearch.query.time.end=2024-02-01T00:00:00Z
```

**Comment by rconway** [comment]
Thanks Lauri. Hadn't spotted that.

**Comment by rconway** [comment, marked as answer]
Fixed in current doc tip - https://github.com/EOEPCA/deployment-guide/commit/f0aefd59b573d660edc1260c27e4e167c7159a68
